eecs485staff/primer-spec
# Enable task-lists and persist the checkbox state

## Context

Currently, task-lists are rendered with disabled checkboxes. This PR enables the checkboxes so students can more easily track their progress. Also, the checkbox state is persisted across page-reloads!

(Initially tracked in https://github.com/eecs485staff/primer-spec/projects/1#card-44682744.)

## Validation

1. Visit the [validation URL](https://preview.seshrs.ml/previews/eecs485staff/primer-spec/72/). Notice that both checkboxes in the "Learning Goals" section are enabled. Check one of the checkboxes and refresh the page. The checkbox should still be checked.

2. Visit the task-list demo page at https://preview.seshrs.ml/previews/eecs485staff/primer-spec/72/demo/task-lists.html. Notice that the checkboxes are enabled and one of them is checked. Check/uncheck the checkboxes and refresh the page. The checkboxes' state should be persisted.

3.  Go back to the index page at the [validation URL](https://preview.seshrs.ml/previews/eecs485staff/primer-spec/72/). Notice that the checkbox state is unaffected by your selections in the demo page.
